kroxylicious-filters/kroxylicious-record-validation/src/test/java/io/kroxylicious/proxy/filter/schema/validation/topic/PerPartitionTopicValidationResultTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter.schema.validation.topic;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.InstanceOfAssertFactories.STREAM;
+
+class PerPartitionTopicValidationResultTest {
+
+    private static final RecordValidationFailure INVALID = new RecordValidationFailure(0, "invalid");
+
+    @Test
+    void empty() {
+        var result = new PerPartitionTopicValidationResult("foo", Map.of());
```

Comment:
I'm struggling with `result` not being the outcome from the actual method being tested (yes I know it is a validationResult and it doesn't really effect the execution of the tests).